test/composites/menu/events/main.js

```diff
@@ -72,9 +73,21 @@ describe('composites/menu/events/main', () => {
       assert.equal(dropdown.getAttribute('aria-expanded'), 'false');
       fire(elements.topBar.querySelector('#drop-1'), 'click');
       setTimeout(() => {
-      assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
+        assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
         done();
-      }, 400); // give animation/other timeouts a change to do stuff
+      }, 400); // give animation/other timeouts a chance to do stuff
+    });
+
+    it('clicking closed dropdown should open that dropdown and close any other open dropdowns', (done) => {
+      const dropdown = elements.dropdown;
+      const dropdown2 = elements.dropdown2;
+      fire(dropdown, 'click');
+      setTimeout(() => {
+        assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
+        fire(dropdown2, 'click');
+        assert.equal(dropdown2.getAttribute('aria-expanded'), 'true');
```

Comment:
Should add an assertion that the first dropdown is now closed after opening the 2nd one